spec/services/draw_starter_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.describe DrawStarter do
+  describe '.start' do
+    it 'calls :start on an instance of DrawStarter' do
+      draw = double('draw')
+      draw_starter = create_mock_draw_starter(draw: draw)
+      DrawStarter.start(draw: draw)
+
+      expect(draw_starter).to have_received(:start)
+    end
+  end
+
+  def create_mock_draw_starter(param_hash)
+    draw_starter = double('draw_starter')
```

Comment:
Yea after reading the post I'm convinced. I'll switch them everywhere in the tests.